src/test/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceTest.java

```diff
@@ -377,4 +384,62 @@ private LinkedGroupDetailsAudit createGroupDetailsAuditEntity(long hearingGroupI
         }
 
     }
+
+    @Nested
+    @DisplayName("getHearingGroup")
+    class GetHearingGroup {
+        public static final String INVALID_REQUEST_NAME = "Invalid Name";
+        public static final String VALID_REQUEST_ID = "Request Name";
+
+        @Test
+        void shouldGetLinkedHearingGroupDetails() {
+            LinkedGroupDetails linkedGroupDetails =
+                generateLinkedGroupDetails(HEARING_GROUP_ID);
+            HearingEntity hearing1 = generateHearingEntity(HEARING_ID1,linkedGroupDetails);
+            HearingEntity hearing2 = generateHearingEntity(HEARING_ID2,linkedGroupDetails);
+
+            given(linkedGroupDetailsRepository.getLinkedGroupDetailsByRequestId(VALID_REQUEST_ID))
+                .willReturn(linkedGroupDetails);
+            given(hearingRepository.findByLinkedGroupId(HEARING_GROUP_ID))
+                .willReturn(List.of(hearing1, hearing2));
+
+            LinkedHearingGroupResponses linkedHearingGroupResponses =
+                service.getLinkedHearingGroupDetails(VALID_REQUEST_ID);
+            GroupDetails returnedGroupDetails = linkedHearingGroupResponses.getGroupDetails();
+            assertFalse(
+                returnedGroupDetails != null
+                    && returnedGroupDetails.getGroupName().isEmpty()
+                    && returnedGroupDetails.getGroupComments().isEmpty());
```

Comment:
Can we split these assertions please:
```suggestion
            assertNotNull(returnedGroupDetails);
            assertFalse(returnedGroupDetails.getGroupName().isEmpty());
            assertFalse(returnedGroupDetails.getGroupComments().isEmpty());
```
Though with the second and third we could be more specific - we know exactly what each of the values should be so should be able to assert each field within the `returnedGroupDetails` with `assertEquals()`?